reference/v8.json

```diff
@@ -163,7 +163,9 @@
     "buffer": {
       "type": "number",
       "default": 128,
-      "doc": "Tile buffer size on each side (measured in 1/512ths of a tile; higher means fewer rendering artifacts near tile edges but slower performance)."
+      "maximum": 512,
```

Comment:
All tiles are normalized to the same size when they are loaded into GL: 512 "units" wide and tall.

The size of a buffer is limited by our representation of a coordinate as a `short`. In practice, this means that buffers cannot be larger than 512 "units." If a tile has a buffer larger than 512 "units" we drop all coordinates outside of the supported range and log warning messages.